source/_lovelace/picture-glance.markdown

```diff
@@ -25,30 +23,80 @@ You can also use `camera` domain entities to use that as image. You can also use
 Screenshot of the picture glance card.
 </p>
 
-**Options**
+{% linkable_title Options %}
 
-| Name | Type | Default | Description
-| ---- | ---- | ------- | -----------
-| type | string | **Required** | `picture-glance`
-| image | string | **Required** | URL of an image
-| entities | list | **Required** | Entity id's
-| navigation_path | string | Optional | Path of URL to use in navigation
-| camera_image | string | Optional | camera domain entity_id 'camera.demo_camera'
-| state_image | object | Optional| See `state_image` object structure.
-| entity | list | Optional | An entity to use for state_image state
-| title | string | Optional | Card title
+{% configuration %}
+type:
+  required: true
+  description: picture-glance
+  type: string
+image:
+  required: true
```

Comment:
image is also optional if using camera_image

Reply:
Presumably then it needs a *one of image, camera_image, or state_image is required* type statement?